### PR TITLE
Add cut-n-paste-support

### PR DESCRIPTION
**Pull request by oven**
Apparently, if  you don't have a menu item, you can't paste using ctrl-v or cmd-v. This pull request adds the menu necessary.

### CODE REVIEW

**Comment by gm-vm** [comment]
Interesting, here the shortcuts work without this.

I tried this anyway, but the menu entries do not work. If I select Cut for example, nothing happens. Shortcuts keep working as usual though.

Out of curiosity, are using OSX?

File: openfortivpn-webview-electron/index.js [hunk 1]
```diff
@@ -91,6 +91,17 @@ app.whenReady().then(() => {
...
+  },{
+    label: "Edit",
+    submenu: [
+        { label: "Undo", accelerator: "CmdOrCtrl+Z", selector: "undo:" },
+        { label: "Redo", accelerator: "Shift+CmdOrCtrl+Z", selector: "redo:" },
+        { type: "separator" },
+        { label: "Cut", accelerator: "CmdOrCtrl+X", selector: "cut:" },
+        { label: "Copy", accelerator: "CmdOrCtrl+C", selector: "copy:" },
+        { label: "Paste", accelerator: "CmdOrCtrl+V", selector: "paste:" },
+        { label: "Select All", accelerator: "CmdOrCtrl+A", selector: "selectAll:" },
```
Comment (gm-vm) [inline openfortivpn-webview-electron/index.js:103]:
```javascript
        { label: "Undo", accelerator: "CmdOrCtrl+Z", role: "undo" },
        { label: "Redo", accelerator: "Shift+CmdOrCtrl+Z", role: "redo" },
        { type: "separator" },
        { label: "Cut", accelerator: "CmdOrCtrl+X", role: "cut" },
        { label: "Copy", accelerator: "CmdOrCtrl+C", role: "copy" },
        { label: "Paste", accelerator: "CmdOrCtrl+V", role: "paste" },
        { label: "Select All", accelerator: "CmdOrCtrl+A", role: "selectAll" },
```
This is actually what works for me. I can't see `selector` in the [documentation](https://www.electronjs.org/docs/latest/api/menu-item).

Does `role` work for you?

**Review by gm-vm — COMMENTED** [review]
Just to answer my previous question, [yes, you are using OSX](https://github.com/gm-vm/openfortivpn-webview/pull/4#issuecomment-1202083852).

**Comment by oven** [comment]
Yes, selector and role work equally well on OS X. Sorry for the delay. Pushed a new commit using role instead :)

**Pull request merged**